apps/api/src/migrations/20220324201352_add_lang_currency.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('UserAccount', (table) => {
+    table.renameColumn('locale', 'language')
+    table.string('currency').notNullable().defaultTo('USD')
+  })
+
+  await knex.schema.createTable('CmsCacheLanguages', (table) => {
+    table.string('code').primary()
+    table.string('name').notNullable()
+    table.string('sort').unique().notNullable()
```

Comment:
I think we want to allow this to be nullable since sort values aren't assigned by default. I ran into an issue locall running this but was able to fix it by making this nullable and changing the sort field for languages in the CMS to "Allow NULL" in the field settings.

Reply:
I opted to add a default to of 1, thanks for the callout!